BambooTracker.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=BambooTracker
```

I don't know alot about .desktop files, so take these comments more as ideas based on examples I've seen than change demands:
```suggestion
Name=BambooTracker
GenericName=OPNA Tracker
GenericName[jp]=OPNAのトラッカーです
```